zookeeper-server/src/main/java/org/apache/zookeeper/server/watch/WatcherCleaner.java

```diff
@@ -102,12 +103,13 @@ public void addDeadWatcher(int watcherBit) {
                 totalDeadWatchers.get() >= maxInProcessingDeadWatchers) {
             try {
                 RATE_LOGGER.rateLimitLog("Waiting for dead watchers cleaning");
-                synchronized(totalDeadWatchers) {
-                    totalDeadWatchers.wait(100);
+                synchronized(processingCompletedEvent) {
+                	processingCompletedEvent.wait(100);
```

Comment:
Can we change this with 4 extra white spaces relative to the synchronized statement?

Reply:
Done